elfpy/utils/apeworx_integrations.py

```diff
@@ -50,7 +54,14 @@ def get_transfer_single_event(tx_receipt: Receipt) -> ContractLog:
         ) from exc
 
 
-def get_pool_state(tx_receipt: Receipt, hyperdrive_contract: ContractInstance):
+def failblog(string: str, *args) -> None:
```

Comment:
get rid of this. If you want to strip the logs down to only errors you can do something like this (on your own end)
```
with open('input_file.txt', 'r') as input_file, open('output_file.txt', 'w') as output_file:
    for line in input_file:
        if 'ERROR' in line:
            output_file.write(line)
```